README.md

```diff
@@ -88,9 +88,9 @@ Tests for Z3-Noodler are located in [src/test/noodler](src/test/noodler).
 
 ## Licensing
 
-The code of Z3-Noodler is distributed under the MIT License. See [LICENSE.md](./LICENSE.md).
+Z3-Noodler is distributed under the MIT License. See [LICENSE.md](./LICENSE.md).
```

Comment:
"licensed" may be better than "distributed", but it does not matter so much

Reply:
Distributed is the standard legalise for this thing. `Licensed` can also be used.

Reply:
Fixed.